src/main/scala/com/fulcrumgenomics/umi/CallMolecularConsensusReads.scala

```diff
@@ -143,7 +145,7 @@ class CallMolecularConsensusReads
 
     // The output file is unmapped, so for now let's clear out the sequence dictionary & PGs
     val outHeader = UmiConsensusCaller.outputHeader(in.header, readGroupId, sortOrder)
-    val out       = SamWriter(output, outHeader, sort=sortOrder)
+    val out       = SamWriter(output, outHeader, sort=sortOrder, async=threads > 1)
```

Comment:
If someone uses `fgbio --async-io=true CallMolecularConsensusReads --threads 1` then this wont be async.  I think that's an issue.

Reply:
Added a check to see if `SamSource.UseAsync` was on as well.